contentblocks/tests/settings/19.py

```diff
@@ -0,0 +1,111 @@
+import os
+from os.path import expanduser
+
+
+USE_TZ = True
+
+TIME_ZONE = "Africa/Johannesburg"
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": "test.db",
+        "USER": "",
+        "PASSWORD": "",
+        "HOST": "",
+        "PORT": "",
+    }
+}
+"""
```

Comment:
Should we remove the commented database? Or are there plans to make use of it for tests?